### DISCUSSION TITLE
Use Editor in readonly mode

### DISCUSSION THREAD

**Discussion by BertrandB21** [adminerevo/adminerevo#192] [Q&A] [answered]
Hello, 
I'm trying to use editor for let users only search in database and prevent deletion. But i Don't see how to change the first colonne with checkbox. And i can(t ind where in src i can change this.
Have you an idea ?

**Comment by d-roman-halliday** [comment, marked as answer]
My first recommendation would be to have editor (or just users) connect with read-only privileges on the user account(s). That will keep the data/data structure safe regardless.

You may have to customise things (creating your own `adminer_object()`), documentation is a bit sparse right now. But you can look into:

- https://docs.adminerevo.org/
- https://www.adminer.org/en/extension/
- https://www.adminer.org/en/plugins/
